src/dataexplorer/dataexplorer-application/src/components/Dashboard/Dashboard.style.scss

```
@@ -1,3 +1,5 @@
+@import '../../variables';
```

Possible to remove '../' here?

```suggestion
@import 'variables';
```

---

I can take a look